scripts/update-sdk.sh

```diff
@@ -17,9 +17,12 @@
 set -euo pipefail
 
 LATEST_SDK_TAG=$(curl -sSfL -X GET  https://api.github.com/repos/mongodb/atlas-sdk-go/releases/latest | jq -r '.tag_name')
-
 LATEST_SDK_RELEASE=$(echo "${LATEST_SDK_TAG}" | cut -d '.' -f 1)
-echo  "==> Updating SDK to latest major version ${LATEST_SDK_TAG}"
+
+echo  "==> Updating SDK to latest major version tag: ${LATEST_SDK_TAG}, release: ${LATEST_SDK_RELEASE}"
+curl -sSfL -X GET  "https://proxy.golang.org/go.mongodb.org/atlas-sdk/${LATEST_SDK_RELEASE}/@v/${LATEST_SDK_TAG}.info"
+GOPROXY=direct
```

Comment:
pick an approach, skipping the proxy or asking the proxy to update

Reply:
WDYT about the latest version, i go for skipping proxy,  but at the end asking the proxy to update so it'll benefit others